roles/kubernetes-apps/ingress_controller/ingress_nginx/templates/role-ingress-nginx.yml.j2

```diff
@@ -32,7 +32,11 @@ rules:
     # Here: "<ingress-controller-leader>-<nginx>"
     # This has to be adapted if you change either parameter
     # when launching the nginx-ingress-controller.
+{% if ingress_nginx_class %}
```

Comment:
I always find strange to have a if/else with the same code..
Why not something like
`resourceNames: ["ingress-controller-leader-{{ ingress_nginx_class | default('nginx') }}"]` and no if/else ?

Reply:
Done